src/main/java/org/springframework/session/data/mongo/ReactiveMongoOperationsSessionRepository.java

```diff
@@ -56,14 +56,14 @@
 
 	private Integer maxInactiveIntervalInSeconds = DEFAULT_INACTIVE_INTERVAL;
 	private String collectionName = DEFAULT_COLLECTION_NAME;
-	private AbstractMongoSessionConverter mongoSessionConverter = new JdkMongoSessionConverter(
-		Duration.ofSeconds(this.maxInactiveIntervalInSeconds));
+	private AbstractMongoSessionConverter mongoSessionConverter;
 
 	private MongoOperations blockingMongoOperations;
 	private ApplicationEventPublisher eventPublisher;
 
-	public ReactiveMongoOperationsSessionRepository(ReactiveMongoOperations mongoOperations) {
+	public ReactiveMongoOperationsSessionRepository(ReactiveMongoOperations mongoOperations, ClassLoader classLoader) {
```

Comment:
This would be a breaking change. Couldn't `ReactiveMongoWebSessionConfiguration` be changed to reuse the existing `mongoSessionConverter` setter?